pkg/ui/distccl/BUILD.bazel

```diff
@@ -3,7 +3,10 @@ load("@io_bazel_rules_go//go:def.bzl", "go_library")
 # TODO (koorosh): genrule produces directory as output, should be changed later.
 genrule(
     name = "genassets",
-    srcs = ["//pkg/ui/workspaces/db-console:db-console-ccl"],
+    srcs = select({
+        "//pkg/ui:with_ui": ["//pkg/ui/workspaces/db-console:db-console-ccl"],
+        "//conditions:default": ["_empty_assets"],
```

Comment:
It is necessary to provide some directory as an input for `genrule` so I've decided to simply create `_empty_assets` and provide it in case build shouldn't include UI.

Reply:
This change breaks the `make` build, right? Because `distoss.go` has `go:embed assets`, but now `assets` is empty (it only has a `.gitkeep` file instead of an `index.html`).

Reply:
Oh yeah, the `make` build is broken. Check the CI results.

Reply:
Fixed by updating `embed` directive in `distoss.go` and `distccl.go` files from:
```
//go:embed assets
```
to
```
//go:embed assets/*
```
now it includes all files including hidden (starting with .)

Reply:
That means that the default `index.html` (the one that says "Binary built without web UI") will no longer be served if people build `cockroach-short` w/ `make`. Is that acceptable? Not sure if anyone on the DB team cares, but UI folks might be confused by the change.